src/assets/locales/de/translations.json

```
@@ -167,10 +167,13 @@
   "contact.error.message": "Deine Nachricht fehlt",
   "contact.error.email": "Deine E-Mail fehlt",
   "contact.error.email-undefined": "Irgendwas stimmt an dieser E-Mail nicht",
-  "contact.action.again-text": "Leider gab es einen Fehler. Bitte versuche es noch einmal. Klappt das nicht, schicke deine Nachricht bitte direkt an <2>beep@satellytes.com</2>",
+  "contact.action.again-text": "Leider gab es einen Fehler. Bitte versuche es noch einmal. Klappt das nicht, schicke deine Nachricht bitte direkt an",
   "contact.action.missing": "Bitte fülle alle benötigten Felder aus",
   "contact.action.send": "Senden",
-  "contact.action.sent": "Gesendet",
+  "contact.action.sent.headline": "E-Mail wurde gesendet",
+  "contact.action.sent.kicker": "Vielen Dank!",
+  "contact.action.sent.info": "Wir schauen uns alles an und melden uns blablabla. Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Etiam porta sem malesuada magna mollis euismod. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Cras justo odio, dapibus ac facilisis in, egestas eget quam.",
```

Here is currently still the Lorem Ipsum text from Figma. It would be nice if someone has a suggestion for this text.

---

Oh I see, we only had the success button before right? I would prefer never check in text like that. I recognize that this comes straight from the design and that they should never do that. Either `lorem ipsum` or a real text. Everything else is not good. Next time you see this you can happil replace it with lorem entirely.

Here my suggestion:

```suggestion
  "contact.action.sent.info": "Vielen Dank für Ihre Nachricht. Wir bearbeiten diese und melden uns schnellstmöglich bei Ihnen.",
```